xwiki-rendering-integration-tests/src/test/java/org/xwiki/rendering/SimpleIntegrationTests.java

```diff
@@ -35,4 +41,17 @@
 @AllComponents
 public class SimpleIntegrationTests
 {
+    @RenderingTestSuite.Initialized
+    public void initialize(MockitoComponentManager componentManager) throws Exception
```

Comment:
I really don't think this in the right place:
* this only cover a subset of the rendering tests
* you can't remove the ExecutionContext after the test execution this way

It would really make much more sense to add before/after code in both RenderingTest right were the code which convert from one syntax to another (and execute tx sometimes) is located.